src/askscript-vscode/package.json

```diff
@@ -1,25 +1,34 @@
 {
-    "name": "askscript-language",
```

Comment:
This file is heavily changed for no good reason. 
I think it's just reformatting. For readability, how about you revert the changes in this file?

Reply:
Yes, it's only a reformatting issue. I executed `npm run prettier`, and inadvertently reformatted reformatted files that were not to intended to be.

**Previous formatting has been restored.**